lean/common.lean

```diff
@@ -276,6 +284,8 @@ def spl := reg8l 4
 def bpl := reg8l 5
 def sil := reg8l 6
 def dil := reg8l 7
+ -- TODO: this order okay?
+def ah  := reg8l 8
```

Comment:
flexdis uses the convention that the high registers are 16 + the low register index.  So ah should be 16. 

I'd write ` def reg8h (i:fin 16) := lhs.reg $ reg.concrete_gpreg (16+i) gpreg_type.reg8l`, then `def ah := reg8h 0`.